packages/labextension/src/index.ts

```diff
@@ -45,7 +45,9 @@ async function activateCondaEnv(
   const model = new CondaEnvironments(settings);
 
   // Request listing available package as quickly as possible
-  Private.loadPackages(model);
+  if (settings?.get('backgroundCaching') ?? true) {
```

Comment:
@danilopeixoto Almost perfect, after the get you need to request `.composite` to get the final value (because the API allows to introspect the difference between user and default values).

```suggestion
  if (settings?.get('backgroundCaching').composite ?? true) {
```